scripts/wordlist.txt

```diff
@@ -144,6 +144,8 @@ Skoda
 SLYR
 SSL
 Survey123
+synchronization
+synchronise
```

Comment:
this should be removed from the wordlist ("synchronise" is the correct spelling of the word and should not have triggered the spellcheck, so there is no need to have in the wordlist)